lib/services/product_service.ex

```diff
@@ -0,0 +1,10 @@
+defmodule EaRestaurantDataLoader.Lib.Service.ProductService do
+  alias EaRestaurantDataLoader.Lib.Documents.Product
+  alias EaRestaurantDataLoader.MongoRepo
+
+  def get_products, do: MongoRepo.all(Product)
+
+  def insert_product(product), do: MongoRepo.insert(product)
```

Comment:
nit: I guess if you insert one product might be the same as inserting many products?

Reply:
Done. It was remove.